### PR TITLE
Move GitHub files to .github directory

### PR DESCRIPTION
**Pull request by ChristophWurst**

### CODE REVIEW

**Review by blizzz — CHANGES_REQUESTED** [review]
Links need to be adjusted, too

```
$ ack CONTRIBUTING 
README.md
64:Please review the [guidelines for contributing](https://github.com/nextcloud/server/blob/master/CONTRIBUTING.md) to this repository.

contribute/HowToApplyALicense.md
44:at [CONTRIBUTING.md][contributing] to learn more how to sign your commits.
46:[contributing]: https://github.com/nextcloud/server/blob/master/CONTRIBUTING.md#sign-your-work
```
```
$ ack issue_template 
.github/CONTRIBUTING.md
22:[template]: https://raw.githubusercontent.com/nextcloud/server/master/issue_template.md
```

**Comment by ChristophWurst** [comment]
@blizzz should be fixed now!

File: build/files-checker.php [hunk 1]
```diff
@@ -54,14 +54,12 @@
...
 	'console.php',
 	'contribute',
-	'CONTRIBUTING.md',
 	'COPYING',
 	'COPYING-README',
 	'core',
 	'cron.php',
 	'index.html',
 	'index.php',
-	'issue_template.md',
```
Comment (MorrisJobke) [inline build/files-checker.php]: Nice one - let me update the release script then as well. 👍

**Review by blizzz — APPROVED** [review]

**Comment by ChristophWurst** [comment]
Squashed and rebased.

**Comment by ChristophWurst** [comment]
Failing test unrelated.

**Review by MorrisJobke — APPROVED** [review]

**Comment by nickvergessen** [comment]
This is quite bad, many apps reference those files:
https://github.com/nextcloud/notifications/blame/master/.github/contributing.md#L20
https://github.com/nextcloud/notifications/blame/master/.github/contributing.md#L25
https://github.com/nextcloud/notifications/blame/master/.github/issue_template.md#L4

and that contributing and issue_template files are basically the same for all "official" apps.

**Comment by ChristophWurst** [comment]
> This is quite bad, many apps reference those files:

Is that all? Then let me file pull requests to those :rocket: 

> and that contributing and issue_template files are basically the same for all "official" apps.

I know, but I fail to see how that's an issue.

**Comment by ChristophWurst** [comment]
Bummer https://github.com/search?q=%22https%3A%2F%2Fgithub.com%2Fnextcloud%2Fserver%2Fblob%2Fmaster%2FCONTRIBUTING.md%22&type=Code

**Comment by nickvergessen** [comment]
Yes, that is exactly what I ment ;)

it's 3+ per app, in about 70+ apps ;)

**Comment by ChristophWurst** [comment]
Ugh, so shall we revert this PR or just move the contributing back?

**Comment by ChristophWurst** [comment]
https://github.com/nextcloud/server/compare/misc/move-back-contribution-md

**Pull request merged**